packages/legacy/cli/src/index.ts

```diff
@@ -1,3 +1,4 @@
+import '@graphql-mesh/include/register-tsconfig-paths';
```

Comment:
Does this respect given `cwd` ?

Reply:
No, it uses process.cwd(). Should it?

Reply:
It is possible to configure cwd with `baseDir` param in the legacy CLI.

Reply:
The programatic tsconfig register supports passing in a cwd, so I'll fix it.
https://github.com/ardatan/graphql-mesh/blob/1a9746f6ca9b517230a0337d5a852bf05707303a/packages/include/src/index.ts#L61

Reply:
Thanks 🙏

Reply:
https://github.com/ardatan/graphql-mesh/pull/7493/commits/4c39b3aa0623c382b6c395c2d3f26a065f38565b